demo_umami_content/tests/src/Functional/UninstallDefaultContentTest.php

```diff
@@ -46,6 +46,20 @@ public function testReinstall() {
     $this->assertEquals(0, $count);
     $module_installer->install(['demo_umami_content']);
     $this->assertRecipesImported($node_storage);
+
+    $block_storage = $this->container->get('entity_type.manager')->getStorage('block_content');
+    $this->assertImportedCustomBlock($block_storage);
+    $module_installer->uninstall(['demo_umami_content']);
```

Comment:
I don't think we need to add more install/uninstall calls, we should be able to slot our tests in with the existing ones.

Reply:
Oh! so you mean to say that blocks should also be tested along with imported nodes ?